devices/Adxl343/ADXL343/Mappings.cs

```diff
@@ -0,0 +1,151 @@
+﻿//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+namespace ADXL343
+{
+    public enum ActInactCtlMap : byte
```

Comment:
please use 1 file per enum where the name of the file is the name of the enum.
In terms of naming convention, we're not using upper case like you do here but readable enums like AcDC, XEnable, YEnable, etc
The only exception would be for internal registers to get close to the specifications (like your current Register one)
And last question: do you need all those enums public? Can some be internal?